feat: add beagle flutter documentation (#717)
ZupIT/beagle-docs: content/en/resources/customization/beagle-for-flutter/services/logger.md

---
title: Logger
weight: 189
description: >-

>   You will find here information of how to use the Logger in Beagle Flutter.

Reviewer: ```suggestion
In this section, you will information on how to use the Logger in Beagle Flutter.
```

---

---

## Introduction

> API for sending log output. Beagle Flutter provides a default implementation called `DefaultEmptyLogger` that doesn't log any information. Here is the contract:

Reviewer: ```suggestion
Logger is an API that sends log output. Beagle Flutter provides a default implementation called `DefaultEmptyLogger` that doesn't log any information. Check out the contract:
```
